README.md

```diff
@@ -30,11 +33,54 @@ and this to your crate root:
 ```rust
 extern crate rand;
 
-// example usage:
-use rand::{Rng, thread_rng};
-let x: f64 = thread_rng().gen();
+use rand::prelude::*;
+
+// basic usage with random():
+let x: u8 = rand::random();
+println!("{}", x);
+
+let y = rand::random::<f64>();
+println!("{}", y);
+
+if rand::random() { // generates a boolean
+    println!("Heads!");
+}
+
+// normal usage needs both an RNG and a function to generate the appropriate
+// type, range, distribution, etc.
+let mut rng = thread_rng();
+if rng.gen() { // random bool
+    let x: f64 = rng.gen(); // random number in range (0, 1)
+    println!("x is: {}", x);
+    let char = rng.gen::<char>(); // Sometimes you need type annotation
+    println!("char is: {}", char);
+    println!("Number from 0 to 9: {}", rng.gen_range(0, 10));
+}
 ```
 
+## Functionality
+
+The Rand crate provides:
+
+- A convenient to use default RNG, `thread_rng`: an automatically seeded,
+  crypto-grade generator stored in thread-local memory.
+- Pseudo-random number generators: `StdRng`, `SmallRng`, `prng` module.
+- Functionality for seeding PRNGs: the `FromEntropy` trait, and as sources of
+  external randomness `EntropyRng`, `OsRng` and `JitterRng`.
+- Most content from `rand_core` (re-exported): base random number generator
+  traits and error-reporting types.
```

Comment:
You want to say this without explaining what `rand_core` is? Not sure whether there should be a section in this README; otherwise maybe link to `https://crates.io/crates/rand_core`.

Reply:
It is linked above?

Reply:
Why not link here?

Reply:
Okay